spec/requests/teachers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'swagger_helper'
+
+def decoded_jwt_token_from_response(response)
+  token_from_request = response.headers['Authorization'].split(' ').last
+  JWT.decode(token_from_request, ENV['DEVISE_JWT_SECRET_KEY'], true)
```

Comment:
Clarification: is the JWT class avaliable because of `json-jwt` gem?

Reply:
yes, I believe so!